eng/pipelines/templates/jobs/new.smoke.tests.yml

```diff
@@ -0,0 +1,33 @@
+parameters:
+  - name: ServiceDirectory
+    type: string
+    default: ''
+  - name: Daily
+    type: boolean
+    default: true
+  - name: TimeoutInMinutes
+    type: number
+    default: 60
+
+jobs:
+  - job: SmokeTest
+    steps:
+      - ${{ if eq(parameters.Daily, true) }}:
+        - task: PowerShell@2
+          displayName: 'Run Nightly SmokeTests'
+          inputs:
+            targetType: 'filePath'
+            filePath: eng/scripts/Smoke_Tests_Daily.ps1
+            arguments: '.'
+            pwsh: true
+
+      - ${{ if eq(parameters.Daily, false) }}:
+        - task: PowerShell@2
+          displayName: 'Run Pre-Release SmokeTests'
+          inputs:
+            targetType: 'filePath'
+            filePath: $(Build.SourcesDirectory)/eng/scripts/Smoke_Tests_Daily.ps1
```

Comment:
Is there a versioning difference here or just a service directory target difference? Should the script just be called "Smoke_Tests.ps1"?

Reply:
I can make that name change